box/blob.go

```diff
@@ -9,50 +9,50 @@ import (
 )
 
 func init() {
-	resources.Add("/TestChainValidationMessageSuiteTestAccountActorCreationfailcreateBLSaccountactorinsufficientbalance", []types.ApplyMessageResult{types.ApplyMessageResult{Receipt: types.MessageReceipt{ExitCode: 2, ReturnValue: []uint8{}, GasUsed: 0}, Penalty: abi.NewTokenAmount(178), Reward: abi.NewTokenAmount(0), Root: "bafy2bzacea7fphntvesdry5ohpz6fd46erq3ea7wx3oimynir73lqjhg7tcc4"}})
```

Comment:
yaaay someone that's not me can do this! :bus:

Reply:
hee hee